core/src/main/scala/org/apache/spark/scheduler/MapStatus.scala

```diff
@@ -118,17 +129,21 @@ private[spark] class CompressedMapStatus(
     MapStatus.decompressSize(compressedSizes(reduceId))
   }
 
+  override def mapTaskId: Long = mapTId
```

Comment:
if we must have a private variable, I'd suggest `def mapTaskId: Long = _mapTaskId`

Reply:
Yep, change all private variable to _mapTaskId in 00e78b2.